src/server/command_registry.cc

```diff
@@ -103,11 +104,24 @@ CommandRegistry& CommandRegistry::operator<<(CommandId cmd) {
     }
     k = it->second;
   }
+
+  family_of_commands_.back().push_back(std::string(k));
+  cmd.SetFamily(family_of_commands_.size() - 1);
+  cmd.SetBitIndex(1ULL << bit_index_++);
```

Comment:
Then its a bit mask 🙂 actually you can store only the index and return 1<<i in get bit mask

Reply:
Yes it's a bit mask! Don't hate me, too lazy to make this change and I don't want to do a shift every time I access it :P (not that it matters though)